Resolutions/XenobladeX_Resolution/rules.txt

```diff
@@ -3,169 +3,230 @@ titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
 name = Resolution
 path = "Xenoblade Chronicles X/Graphics/Resolution"
 description = Changes the resolution of the game. Made by getdls.
-version = 4
+version = 5
 
-
-[Preset]
-name = 1280x720 (Default)
+[Default]
 $width = 1280
 $height = 720
 $gameWidth = 1280
 $gameHeight = 720
-$internalRes = 1.0
-$dither = 0.01
-$scaleShader = 1.0
-$scaleBlur = 0.05
+$internalScale = 1.0
+$shadowScale = 1.0
+$skellViewScale = 1.0
 
-// Quality
 
+# SCREEN RESOLUTION
```

Comment:
Swap this line with the empty line above it.

Reply:
Reformatted.